.travis.yml

```diff
@@ -132,8 +138,6 @@ jobs:
         - make image-cephcsi || travis_terminate 1;
         - scripts/travis-helmtest.sh v1.17.0 || travis_terminate 1;
 
-deploy:
-  - provider: script
-    on:  # yamllint disable-line rule:truthy
-      all_branches: true
-    script: ./deploy.sh
+    - stage: deploy
+      name: push container images to quay.io
```

Comment:
need to change the name. it also pushes the helm charts

Reply:
Sure, renamed to 'push artifacts to repositories'